Assets/Mirror/Editor/NetworkScenePostProcess.cs

```diff
@@ -8,6 +8,50 @@ namespace Mirror
 {
     public class NetworkScenePostProcess : MonoBehaviour
     {
+        // Coburn: This runs to ensure people haven't done things like
+        // attempting to put a NetworkIdentity on a NetworkManager, or
+        // NetworkBehaviours on a NetworkManager.
+        [PostProcessScene]
+        public static void DetectWrongUsageOfNetworkBehavioursAndIdentities()
+        {
+            IEnumerable<NetworkManager> netManagers = Resources.FindObjectsOfTypeAll<NetworkManager>()
+                .Where(netManager => netManager.gameObject.hideFlags != HideFlags.NotEditable &&
+                                   netManager.gameObject.hideFlags != HideFlags.HideAndDontSave &&
+                                   netManager.gameObject.scene.name != "DontDestroyOnLoad" &&
+                                   !PrefabUtility.IsPartOfPrefabAsset(netManager.gameObject));
+
+            int netManCount = netManagers.Count();
+            if (netManCount > 0)
+            {
+                if (netManCount > 1)
+                {
+                    // Very bad idea to have more than one NetworkManager.
+                    Debug.LogError("Multiple NetworkManagers detected in scene, this may cause problems!");
+                }
+
+                foreach (NetworkManager netManager in netManagers)
+                {
+                    // NetworkBehaviour check.
+                    NetworkBehaviour[] checkForNetBehaviours = netManager.gameObject.GetComponents<NetworkBehaviour>();
```

Comment:
```suggestion
                    NetworkBehaviour[] checkForNetBehaviours = netManager.gameObject.GetComponentsInChildren<NetworkBehaviour>();
```

Reply:
Interesting - did not know that, I thought it would only grab them in children instead of including the parent.